packages/webapp/components/layouts/MainFeedPage.tsx

```diff
@@ -51,9 +51,7 @@ export default function MainFeedPage({
       router.replace('/');
     } else if (isFinderPage) {
       setIsSearchOn(true);
-      if (!feedName) {
-        setFeedName('popular');
-      }
+      setFeedName('search');
```

Comment:
Just checking, I've seen this too and wasn't sure why it was like this, so just making sure we've checked this properly

Reply:
Yep. This is a bug fix.